tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -25,7 +28,26 @@ def setUpClass(cls):
         """Init single organization, product and repository for all tests"""
         super(ContentViewFilterTestCase, cls).setUpClass()
         cls.org = make_org()
-        cls.product = make_product({u'organization-id': cls.org['id']})
+        product_name = gen_string('alpha')
+        try:
+            cls.product = make_product({
+                u'organization-id': cls.org['id'],
+                u'name': product_name,
+            })
+        except CLIFactoryError:
```

Comment:
there is no bz yet, since this issue is not reproducible anywhere except jenkins :/